sdk/spring/spring-cloud-azure-actuator/pom.xml

```diff
@@ -105,6 +105,18 @@
       <version>12.16.0</version> <!-- {x-version-update;com.azure:azure-storage-queue;dependency} -->
       <optional>true</optional>
     </dependency>
+    <dependency>
+       <groupId>com.azure.spring</groupId>
+       <artifactId>spring-cloud-azure-appconfiguration-config-web</artifactId>
+       <version>5.0.0-beta.1</version><!-- {x-version-update;com.azure.spring:spring-cloud-azure-appconfiguration-config-web;current} -->
+      <optional>true</optional>
+     </dependency>
+     <dependency>
+       <groupId>com.azure.spring</groupId>
+       <artifactId>spring-cloud-azure-appconfiguration-config</artifactId>
+       <version>5.0.0-beta.1</version><!-- {x-version-update;com.azure.spring:spring-cloud-azure-appconfiguration-config-web;current} -->
```

Comment:
Wrong artifact id here.

Reply:
Why this artifact not exist in main branch? https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/spring/spring-cloud-azure-actuator/pom.xml

Reply:
I think this has to do with me adding module-info, for some reason it can't be found unless I explicitly say it, even though the web version has it too.